sass/styles/app.scss

```diff
@@ -104,6 +104,11 @@ blockquote {
     p:last-of-type {
         margin-bottom: 0;
     }
+
```

Comment:
Remark: for reasons I don't really know, `p:last-of-type` above is load-bearing, if I comment that out, blockquote resorts to italic text 🤷